index.html

```diff
@@ -29,49 +29,97 @@ <h1 id="title">Virtual Gulingtangan</h1>
 
     <main>
       <div class="pots-container">
-        <div class="pots empty"></div>
+        <div class="pots empty">
+          <audio class="notes">
+          </audio>
+        </div>
         <div class="pots" id="Db4">
           <img src="./assets/gulingTanganSingle.svg" alt="" srcset="" />
+          <audio class="notes">
+            <source src="./assets/gulingtangan-notes/Db4.wav">
```

Comment:
Instead of adding the audio tag one by one arah html. Try to make use of javascript instead. The hint is by using the id tag which also has the desire note to be added

Reply:
Also the problem with this implementation is that, you have to wait until the audio is finished, to be played again. Whenever the person clicks or press a key, they should always create _new Audio()_